lib/importproject.cpp

```diff
@@ -186,9 +186,12 @@ ImportProject::Type ImportProject::import(const std::string &filename, Settings
     mPath = Path::getPathFromFilename(Path::fromNativeSeparators(filename));
     if (!mPath.empty() && !endsWith(mPath,'/'))
         mPath += '/';
+    if (mPath.empty())
+        mPath = std::string("./");
```

Comment:
This is unnecessary - you can simply assign a literal to a `std::string` object.

Reply:
Fixed